tests/Geppetto/test/campaigns/clients/common_client.js

```diff
@@ -0,0 +1,386 @@
+  }
+
+  async isVisible(selector, wait = 0, options = {}) {
+    await this.waitFor(wait, options);
+    global.visible = await page.$(selector) !== null;
+  }
+
+  async getBaseUrl(globalVariable) {
+    global.tab[globalVariable] = await page.url();
+  }
+
+  async switchShop(isSelected = 'All shop') {
+    await this.waitForAndClick(Menu.dashboard_menu_list);
+    await this.waitForAndClick(DashBoardPage.shopname_button, 1000);
+    await page.waitFor(1000);
+    let index = 1;
+    if (isSelected !== 'All shop') {
+      index = await isSelected === 'First shop' ? 3 : 4;
+    }
+    await page.waitFor(2000);
+    await this.waitForAndClick(DashBoardPage.shopname_option.replace('%ID', index));
+    await page.waitFor(3000);
+  }
+
+  stringifyNumber(number) {
+    let special = ['zeroth', 'first', 'second', 'third', 'fourth', 'fifth', 'sixth', 'seventh', 'eighth', 'ninth', 'tenth', 'eleventh', 'twelfth', 'thirteenth', 'fourteenth', 'fifteenth', 'sixteenth', 'seventeenth', 'eighteenth', 'nineteenth'];
+    let deca = ['twent', 'thirt', 'fort', 'fift', 'sixt', 'sevent', 'eight', 'ninet'];
+    if (number < 20) return special[number];
+    if (number % 10 === 0) return deca[Math.floor(number / 10) - 2] + 'ieth';
+    return deca[Math.floor(number / 10) - 2] + 'y-' + special[number % 10];
+  }
+
+  async closeWindow() {
+    await page.close();
+  }
+
+  async getSelectedValue(selector, value) {
+    await page.waitForSelector(selector, {timeout: 90000});
+    global.selectedValue = await page.select(selector, value);
+
+  }
+
+  async checkSelectedValue(selector, textToCheckWith, wait = 0) {
+    await this.waitFor(wait);
+    await this.waitFor(selector);
+    await page.$eval(selector + ' > option[selected]', el => el.innerText).then((text) => expect(text).to.equal(textToCheckWith));
+  }
+
+  async checkValidationInput(selector, textToCheckWith) {
+    await this.waitFor(selector);
+    await page.$eval(selector, (el) => el.title).then((title) => {
+      expect(title).to.equal(textToCheckWith);
+    });
+  }
+
+  async dragAndDrop(sourceSelector, destinationSelector) {
+    let firstElement = await page.$(sourceSelector);
+    let secondElement = await page.$(destinationSelector);
+    let first_bounding_box = await firstElement.boundingBox();
+    let second_bounding_box = await secondElement.boundingBox();
+    let mouse = page.mouse;
+    await mouse.move(first_bounding_box.x + (first_bounding_box.width / 2), first_bounding_box.y + (first_bounding_box.height / 2));
+    await mouse.down();
+    await mouse.move(second_bounding_box.x + (second_bounding_box.width / 2), second_bounding_box.y + (second_bounding_box.height / 2));
+    await mouse.up();
+  }
+
+  async getTextInVar(selector, globalVar, wait = 0, split = false) {
+    if (split) {
+      await this.waitFor(wait);
+      await this.waitFor(selector);
+      await page.$eval(selector, el => el.innerText).then((text) => {
+        global.tab[globalVar] = text.split(': ')[1];
```

Comment:
Ony this line change.
You can refactor the entire function by removing duplicate and adding `split ? text.split(': ')[1] : text;`